### DISCUSSION TITLE
Data Export

### DISCUSSION THREAD

**Discussion by sickmeyer** [cloudscape-design/components#1325] [Q&A] [answered]
Hello, 

Is there an ability to export any chart data to CSV, PNG?

Thanks!
-josh

**Comment by rubencarvalho** [comment, marked as answer]
Hello Josh,

We currently don't natively support this out-of-the-box and we don't have short-term plans to implement an 'exporting feature' for our charts.
We did take a note for your request and will use it as a data point when planning future charts enhancements.

We will keep this discussion open to collect any additional feedback or use cases.

Thank you.
